### PR TITLE
minor updates to Before You Begin (MC doc).

### PR DESCRIPTION
**Pull request by ericvn**
[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/en/docs/setup/install/multicluster/before-you-begin/index.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ In addition, review the requirements and perform the initial steps below.
 ### Cluster
 
 This guide requires that you have two Kubernetes clusters with any of the
-[supported Kubernetes versions](/docs/setup/platform-setup).
+supported Kubernetes versions: {{< supported_kubernetes_versions >}}.
```
Comment (frankbu) [inline content/en/docs/setup/install/multicluster/before-you-begin/index.md:21]: I had just noticed that the Getting Started page could also use something like this. Can you change that one to something like this:
```
These steps require you to have a {{< gloss >}}cluster{{< /gloss >}} running a
compatible version of Kubernetes ({{< supported_kubernetes_versions >}}). You can use ...
```

File: content/en/docs/setup/install/multicluster/before-you-begin/index.md [hunk 2]
```diff
@@ -59,7 +59,7 @@ Istio may change slightly.
 
 This guide will assume that you use a common root to generate intermediate
 certificates for each cluster. Follow the [instructions](/docs/tasks/security/cert-management/plugin-ca-cert/)
-to generate and push a ca certificate secrets to both the `cluster1` and `cluster2`
+to generate and push a ca certificate secret to both the `cluster1` and `cluster2`
```
Comment (frankbu) [inline content/en/docs/setup/install/multicluster/before-you-begin/index.md:62]:
```suggestion
to generate and push a CA certificate and secret to both the `cluster1` and `cluster2`
```

File: content/en/docs/setup/install/multicluster/before-you-begin/index.md (hunk 1 above)
Comment (ericvn) [reply content/en/docs/setup/install/multicluster/before-you-begin/index.md:21]: Updated.

File: content/en/docs/setup/install/multicluster/before-you-begin/index.md (hunk 2 above)
Comment (ericvn) [reply content/en/docs/setup/install/multicluster/before-you-begin/index.md]: Updated spelling but I think the key piece that needs to be in the cluster is the secret (which was made from the cert files). I'm understand that secret to be called the `CA certificate secret`.

**Review by frankbu — APPROVED** [review]

**Pull request merged**